plugin/pkg/scheduler/factory/factory.go

```diff
@@ -389,7 +389,8 @@ func (f *ConfigFactory) CreateFromKeys(predicateKeys, priorityKeys sets.String,
 	}
 
 	f.Run()
-	algo := core.NewGenericScheduler(f.schedulerCache, predicateFuncs, predicateMetaProducer, priorityConfigs, priorityMetaProducer, extenders)
+	// TODO(harryz) use equivalence cache instead of nil here when #36238 get merged
```

Comment:
update this TODO , now that we have the equiv cache, i assume resour will be implementing the ecache?